webrtc.html

```diff
@@ -6082,6 +6082,63 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               <a><code>RTCRtpTransceiver</code></a>. The sender of this
               transceiver will no longer send, the receiver will no longer
               receive, and the <a>negotiation-needed flag</a> is set.</p>
+              <p>Stopping a transceiver will cause future calls to
+              <code>createOffer</code> to generate a zero port in the
+              <a>media description</a> for the corresponding transceiver,
+              as defined in <span data-jsep="stop">[[!JSEP]]</span>.</p>
+              <p>When this method is invoked, the User Agent MUST run the
+              following steps:</p>
+              <ol>
+                <li>
+                  <p>Let <var>transceiver</var> be the 
+                  <code><a>RTCRtpTransceiver</a></code> object which is to
+                  be stopped.</p>
+                </li>
+                <li>
+                  <p>Let <var>connection</var> be the
+                  <code><a>RTCPeerConnection</a></code> object on which
+                  the <var>transceiver</var> is to be stopped.</p>
+                </li>
+                <li>
+                  <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                  <code>true</code>, throw an <code>InvalidStateError</code>
+                  exception and abort these steps.</p>
+                </li>
```

Comment:
Do we want an early return for if the transceiver is already stopped? At least, setting the negotiation-needed flag should be avoided in that case.